### PR TITLE
[UXE-5149] fix: service keeps loading when there is no service registered

### PR DESCRIPTION
**Pull request by pauloSF0**
### Explain what was fixed and the correct behavior.
fix: service keeps loading when there is no service registered

### Does this PR introduce UI changes? Add a video or screenshots here.

https://github.com/user-attachments/assets/0424aa4a-ea71-4b17-8c52-76d781a39b50

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] bug: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [x] Application responsiveness was tested to different screen sizes
- [x] New tests are added to prevent the same issue from happening again
- [ ] UI changes are validated by a team designer
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

**Review by maxwelbm — APPROVED** [review]

**Review by peterpaulo-azion — DISMISSED** [review]

**Review by lucasmendes21 — DISMISSED** [review]

File: src/views/EdgeNode/FormFields/FormFieldsDrawerService.vue [hunk 1]
```diff
@@ -4,22 +4,44 @@
...
+
+  const getServices = async () => {
+    try {
+      loadingOptionsServices.value = true
+      listService.value = await props.listServicesHandle({
+        edgeNodeId: props.edgeNodeId,
+        bound: props.bound
+      })
+    } finally {
+      loadingOptionsServices.value = false
```
Comment (william-tome) [inline src/views/EdgeNode/FormFields/FormFieldsDrawerService.vue:39]: Não estamos tratando o erro.

File: src/views/EdgeNode/FormFields/FormFieldsDrawerService.vue [hunk 2]
```diff
@@ -34,6 +56,8 @@
   const theme = computed(() => {
     return store.currentTheme === 'light' ? 'vs' : 'vs-dark'
   })
+
+  getServices()
```
Comment (william-tome) [inline src/views/EdgeNode/FormFields/FormFieldsDrawerService.vue:60]: Move isso pra dentro de um hook (onMounted / onBeforeMount)

Comment (pauloSF0) [reply src/views/EdgeNode/FormFields/FormFieldsDrawerService.vue]: add onMounted

File: src/views/EdgeNode/FormFields/FormFieldsDrawerService.vue (hunk 1 above)
Comment (pauloSF0) [reply src/views/EdgeNode/FormFields/FormFieldsDrawerService.vue]: add toast

File: src/views/EdgeNode/Drawer/index.vue [hunk 3]
```diff
@@ -135,8 +123,10 @@
   >
     <template #formFields="{ disabledFields }">
       <FormFieldsDrawerService
-        :listServices="listService"
+        :edgeNodeId="props.edgeNodeId"
```
Comment (lucasmendes21) [inline src/views/EdgeNode/Drawer/index.vue:126]: Nesse caso do uso de um props no template não é necessário o uso do props, você consegui acessar direto ex: :edgeNodeId="edgeNodeId"

File: src/views/EdgeNode/Drawer/index.vue [hunk 4]
```diff
@@ -135,8 +123,10 @@
...
+        :listServicesHandle="props.listServiceEdgeNodeService"
         :disabledFields="disabledFields"
+        :bound="true"
```
Comment (lucasmendes21) [inline src/views/EdgeNode/Drawer/index.vue:129]: nesse caso ao passar uma propriedade como true, passa apenas o nome dela que po default já é true ex: bound

Comment (pauloSF0) [reply src/views/EdgeNode/Drawer/index.vue]: done

**Review by lucasmendes21 — APPROVED** [review]

**Review by william-tome — APPROVED** [review]

**Pull request merged**